atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/StreamTestSchema.java

```diff
@@ -59,6 +59,12 @@ private static Schema generateSchema() {
                     .isAppendHeavyAndReadLight()
                     .build());
 
+        // streams can have up to Integer.MAX_VALUE as the inMemoryThreshold
+        schema.addStreamStoreDefinition(
+                new StreamStoreDefinitionBuilder("stream_test_max_mem", "stream_test_max_mem", ValueType.VAR_LONG)
+                    .inMemoryThreshold(Integer.MAX_VALUE)
```

Comment:
We probably just want to defer to this limit, although it is evidently not used everywhere.